misc/whisper_pod_transcriber/api.py

```diff
@@ -103,7 +102,7 @@ async def poll_status(call_id: str):
         function_call.get(timeout=0.1)
     except TimeoutError:
         pass
-    except modal.exception.RemoteError as exc:
+    except Exception as exc:
```

Comment:
Hmm, so apparently you don't _always_ get a `RemoteError`: 

<img width="1073" alt="image" src="https://user-images.githubusercontent.com/12058921/200199908-84342665-17e8-4114-8eac-63bc57e579ab.png">